ci/envs/38-no-optional-deps.yaml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - python=3.8
   # required
   - pandas
+  - rtree
```

Comment:
```suggestion
```

rtree is an optional dependency. This env should not have it.

Reply:
Done.